Cargo.toml

```diff
@@ -23,8 +23,12 @@ weak = []
 internal-test-strategies = []
 # Possibly some strategies we are experimenting with. Currently empty. No stability guarantees are included about them.
 experimental-strategies = []
+# no_std doesn't have RwLock
+no_std = ["parking_lot"]
```

Comment:
The cargo recommendation for features is that they be additive, not subtractive: https://doc.rust-lang.org/cargo/reference/features.html#feature-unification

Specifically this would imply a std= feature, make the default features include std; though sadly cargo doesn't support managing the dependency properly : https://github.com/rust-lang/cargo/issues/1839.

Perhaps a comment referencing that issue?